cfme/optimize/planning.py

```diff
@@ -0,0 +1,102 @@
+from navmazing import NavigateToAttribute
+from widgetastic.widget import View
+from widgetastic_patternfly import Tab
+
+from cfme.base.login import BaseLoggedInPage
+from cfme.modeling.base import BaseCollection
+from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep
+from widgetastic_manageiq import (BootstrapSelect, Button, Checkbox, Input, LineChart,
+                                  SummaryFormItem, Table)
+
+
+class PlanningFilter(BaseLoggedInPage):
```

Comment:
Please use a name like `PlanningFilterView` or `PlanningFilterForm` so as not to confuse the class with an object modeling a filter itself. (Where I would expect attributes representing a PlanningFilter